pkgs/development/ocaml-modules/nice_parser/default.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildDunePackage,
+  fetchFromGitHub,
+  menhir,
+  ppx_jane,
+  base,
+  stdio,
+  odoc,
```

```suggestion
  stdio,
```

The unused arguments can be removed, and some of the intermediate commits squashed into a single one.